global.json

```diff
@@ -1,5 +1,4 @@
 {
   "sdk": {
-    "version": "2.1.403"
```

Comment:
Instead of removing this, can you update it to `2.1.503`? 

It appears that the easy to follow instructions for the linux dotnet sdk installs that version. My goal is to see how our PR builds behave when this is done.